7024Movies/Pages/DiscoverThriller.cshtml.cs

```diff
@@ -52,11 +52,11 @@ public void OnGet()
                     {
                         //k_3re1w44s
                         //k_vxcfqztc
-                        string movieid = movie.ImdbId.ToString();
-                        uri = String.Format("https://imdb-api.com/en/API/Title/k_3re1w44s/{0}", movieid);
+                        string MovieId = movie.ImdbId.ToString();
+                        uri = String.Format("https://imdb-api.com/en/API/Title/k_3re1w44s/{0}", MovieId);
                         string ImdbString = webClient.DownloadString(uri);
-                        var imdbrating = IMDB.MovieImdb.FromJson(ImdbString);
-                        thrillerimdb.Add(imdbrating);
+                        var ImdbRating = IMDB.MovieImdb.FromJson(ImdbString);
+                        thrillerimdb.Add(ImdbRating);
```

Comment:
By this best practices article it talks about using Carmelcase when declaring variables
https://dev.to/prahladyeri/underscores-camelcasing-and-pascalcasing-the-three-naming-conventions-every-programmer-should-be-aware-of-3aed